### PR TITLE
GHC 7.10.2

### PR DESCRIPTION
**Pull request by emptyflask**
Unable to test right now due to the incredibly long build process. Hopefully it works.

### CODE REVIEW

File: Library/Formula/ghc.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.1/ghc-7.10.1-src.tar.xz"
-  sha256 "92f3e3d67a637c587c49b61c704a670953509eb4b17a93c0c2ac153da4cd3aa0"
+  url "https://downloads.haskell.org/~ghc/7.10.2/ghc-7.10.2-src.tar.xz"
+  sha256 "54cd73755b784d78e2f13d5eb161bfa38d3efee9e8a56f7eb6cd9f2d6e2615f5"
 
   revision 1
```
Comment (dunn) [inline Library/Formula/ghc.rb]: Revision lines need to be reset with new versions.

**Comment by DomT4** [comment]
I've *massively* bumped the test bot's timeout here, but if the build queue gets too backed up I'll likely kill the build until the early hours. We'll see how it goes.

File: Library/Formula/ghc.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.1/ghc-7.10.1-src.tar.xz"
-  sha256 "92f3e3d67a637c587c49b61c704a670953509eb4b17a93c0c2ac153da4cd3aa0"
+  url "https://downloads.haskell.org/~ghc/7.10.2/ghc-7.10.2-src.tar.xz"
+  sha256 "54cd73755b784d78e2f13d5eb161bfa38d3efee9e8a56f7eb6cd9f2d6e2615f5"
 
-  revision 1
+  revision 0
```
Comment (dunn) [inline Library/Formula/ghc.rb]: Sorry, I wasn't very clear.  `revision 0` is the default, so this line can be omitted altogether.

(Pushing another change will trigger a new build, though, so you can wait to fix this until we see if anything else needs changing.)

Comment (DomT4) [inline Library/Formula/ghc.rb]: Let's just do it now. I'd prefer to only have to do one build on this if we possibly can. I can keep kicking them out of the queue until we're happy.

Comment (emptyflask) [inline Library/Formula/ghc.rb]: Sorry about that. I'll push another commit in a second.

**Comment by markandrus** [comment]
@emptyflask thanks, was just wondering if anyone was working on this. I tried a similar formula before lunch, but it had been running for seemingly too long and I assumed I did something wrong...

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/aa76c29d91621cb151f85141ac7ce1d9cd1787ab. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**